newa/cli.py

```diff
@@ -194,50 +193,99 @@ def cmd_event(ctx: CLIContext, errata_ids: list[str]) -> None:
 def cmd_jira(ctx: CLIContext, issue_config: str) -> None:
     ctx.enter_command('jira')
 
-    # this is here temporarily so we generate fake Jira issue IDs
-    jira_id_gen = itertools.count(start=1)
+    # Get control of Jira.
+    # TODO: Hardcoded project for now.
+    # It will be served from the component-config.yaml later.
+    jira = JiraHandler("BASEQESEC")
```

Comment:
I think it would help if we could get the config related PR in and adjust this one according to it. It would also make testing for users way more comfortable.

Reply:
Agreed, I'll keep it in the current one and once we have config PR merged, I'll add jira project and project-specific fields/statuses mappings into the configuration.

Reply:
Fixed in the current PR.